### PR TITLE
Plugin package

### PR DESCRIPTION
**Pull request by br-lewis**
Adds a `make plugin` target to build both the Go and Python CLIs then packages them both into a plugin ZIP in the build directory.

### CODE REVIEW

**Comment by ArmandGrillet** [comment]
@mesosphere-ci run all integration tests

File: scripts/plugin/package_plugin.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+from distutils import dir_util, file_util
+
+plugin_toml_template = '''
+schema_version = 1
+name = "dcos-core-cli"
+
+[[commands]]
+name = "job"
+path = "bin/dcos{0}"
```
Comment (rgo3) [inline scripts/plugin/package_plugin.py]: For now, until we decide the go rewrite is ready, I would keep the python binary.

Comment (br-lewis) [reply scripts/plugin/package_plugin.py]: 👍
This brings up the bigger point of whether or not I should be packaging the Go part if nothing is using it. I think I'll take it out for now.

File: completion/bash/job.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+_dcos_job() {
```
Comment (bamarni) [inline completion/bash/job.sh]: Any reason for adding an empty script for job?

**Comment by br-lewis** [comment]
@mesosphere-ci run all integration tests

**Comment by ArmandGrillet** [comment]
@br-lewis Do you want me to squash and merge or do you want to edit your branch so that I can rebase and merge?

**Comment by ArmandGrillet** [comment]
@mesosphere-ci run all integration tests

**Comment by br-lewis** [comment]
@mesosphere-ci run all integration tests

**Pull request merged**